CSharp/Library/Dialogs/PromptDialog.cs

```diff
@@ -159,7 +159,7 @@ public PromptConfirm(string prompt, string retry, int attempts)
 
             protected override bool TryParse(Message message, out bool result)
             {
-                switch (message.Text)
+                switch (message.Text.Trim().ToLower())

```

Comment:
I think it is safer to use ToLowerInvariant() but since we are not localized, it doesn't make a difference.